### ISSUE TITLE
show Input field sets at the top?

### ISSUE THREAD

**Issue by sivagopalvotigo** [kevinchappell/formBuilder#522]
### Description:
I want to show Input field sets at the top instead of bottom is there any option to do that.
### Environment Details:

 - formBuilder Version: 2.2.5
 - Browser: chrome
 - OS: ubuntu

### Screenshot - *(optional)*
![screenshot from 2017-06-05 18 09 55](https://cloud.githubusercontent.com/assets/28002719/26784305/8f334618-4a1a-11e7-8ad4-8e831dece49f.png)

**Comment by gotounix** [comment]
```
var options = {
      controlOrder: [
        'text',
        'textarea'
      ]
    };
$(container).formBuilder(options);
```

**Comment by sivagopalvotigo** [comment]
Those are not default input fields those are custom input sets.

**Comment by sivagopalvotigo** [comment]
It got fixed when upgraded to version 2.5.1
